diff -uNr krb5-auth-dialog-0.13.orig/src/krb5-auth-dialog.c krb5-auth-dialog-0.13/src/krb5-auth-dialog.c
--- krb5-auth-dialog-0.13.orig/src/krb5-auth-dialog.c	2015-04-07 11:54:03.720249778 +0800
+++ krb5-auth-dialog-0.13/src/krb5-auth-dialog.c	2015-04-07 11:56:19.214686582 +0800
@@ -205,6 +205,7 @@
 	krb5_creds my_creds;
 	krb5_timestamp now;
 	gboolean retval = FALSE;
+	memset (&my_creds, 0, sizeof (my_creds));
 
 	ka_applet_set_tgt_renewable(applet, FALSE);
 	if (!ka_get_tgt_from_ccache (kcontext, &my_creds)) {
@@ -811,6 +812,7 @@
 	krb5_ccache ccache;
 	krb5_get_init_creds_opt opts;
 
+	memset (&my_creds, 0, sizeof (my_creds));
 	if (kprincipal == NULL) {
 		retval = ka_parse_name(applet, kcontext, &kprincipal);
 		if (retval)
